peps/pep-0757.rst

```diff
@@ -4,11 +4,12 @@ Author: Sergey B Kirpichev <skirpichev@gmail.com>,
         Victor Stinner <vstinner@python.org>
 PEP-Delegate: C API Working Group
```

Comment:
Has the SC confirmed the WG delegation for this PEP?

https://github.com/python/steering-council/issues/259 says this PEP is special, and in general there is no standing delegation, but I don't see a clear decision regarding this PEP.

cc @Yhg1s

Reply:
PEP 757 was created from https://github.com/capi-workgroup/decisions/issues/35 : it's a working document for the C API Working Group to help to take a decision. I created PEP 757 with `PEP-Delegate: C API Working Group`.

Reply:
There was a misunderstanding on my side about the C API WG ability to handle their own PEPs.

I submitted [PEP 757 to the Steering Council](https://github.com/python/steering-council/issues/264).